### DISCUSSION TITLE
How to add horizontal margin to confirm, spin, choose... ?

### DISCUSSION THREAD

**Discussion by KristhianX** [charmbracelet/gum#141] [Q&A] [answered]
Hi, I'm very happy replacing parts of my script with gum. But I will like to add a little horizontal padding or margin to some elements (gum spin, choose, the confirm prompt) to match the style I was trying to get. Is there any way?

I added a space before the prompt in gum input, but I have no idea how to move the gum spin.

Thanks in advance! Great work!

**Comment by maaslalani** [comment, marked as answer]
Generally, you need to use the `--<>.margin` flag to do this, for example on `gum spin` you can add a left margin with the following:

```
gum spin --spinner.margin "0 0 0 5" -- sleep 1
```

<img width="610" alt="image" src="https://user-images.githubusercontent.com/42545625/188370529-9f7368a1-a655-446a-82a6-036aba436c26.png">

**Reply by KristhianX** [reply]
Thank you!
